src/screens/SettingsScreen.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+import * as React from 'react';
+import { inject, observer } from 'mobx-react';
+import { Text, View, Switch, StyleSheet } from 'react-native';
+import {
+  Body,
+  Container,
+  Content,
+  Header,
+  Left,
+  List,
+  ListItem,
+  Right,
+  Title,
+} from 'native-base';
+
+@inject('appSettingsStore')
+@observer
+class SettingsScreen extends React.Component<Props> {
+  static navigationOptions = {
+    drawerLabel: 'Settings',
+  };
+
+  render(): React.Element<*> {
+    return (
+      <Container>
+        <Header>
```

Comment:
And yeah..I have to use native-base header insetad react-navigation header since we have drawerRouter inside stackRouter, and only stack router can have header component... So in the current routes structure with react-navigation header the drawer will be below header. So we can use native-base header or wrap `every` drawer route in stackRouter itself...I choose the first solution.